doc/howto/config.rst

```diff
@@ -437,6 +437,22 @@ Example::
     }
 
 
+allow_unsolicited
+"""""""""""""""""
+
+When set to true, IdP initiated SSO is allowed. This means that a user can
+start an authentication flow from the IdP, and once the user is authenticated
```

Comment:
This is a little misleading, the user normally selects an SP that they want to access before they authenticate to the IdP. 

I would rephrase that to : 
This means that this SP will consume unsolicited SAML Responses, i.e. SAML Responses for which it has not sent a respective SAML Authentication Request.